watson-transcribe.py

```diff
@@ -49,9 +50,11 @@ def main():
 
     with open(args.file, 'rb') as audio_file:
         print("Sending audio to watson to recognize, this will take 30+ minutes")
+        print("Using custom model {0}".format(args.customization))
         print("Please be patient and don't kill this process while running")
         output = speech_to_text.recognize(
             audio_file, content_type='audio/flac', timestamps=True,
+            customization_id=args.customization,
```

Comment:
Yes, the only issue is the print statement looks a bit weird: 

```
Sending audio to watson to recognize, this will take 30+ minutes
Using custom model None
Please be patient and don't kill this process while running
```